src/types/sbitstream.jl

```diff
@@ -39,7 +39,7 @@ SBitstream{T}(value::Real) where {T<:Real} = SBitstream(convert(T, value))
 Base.convert(::Type{SBitstream{T}}, s::SBitstream) where {T<:Real} =
     SBitstream{T}(s.bits, convert(T, s.value), s.id)
 
-const SBitstreamLike = Union{<:SBitstream, VecOrMat{<:SBitstream}}
+    const SBitstreamLike = Union{<:SBitstream, VecOrMat{<:SBitstream}, Array{<:SBitstream}}
```

Comment:
Let's just make it really generic.

```suggestion
const SBitstreamLike = Union{<:SBitstream, AbstractArray{<:SBitstream}}
```